src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -43,7 +48,7 @@ namespace Microsoft.ML.Transforms.Text
     // See <a href="https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.TestFramework/DataPipe/TestDataPipe.cs"/>
     // for an example on how to use LdaTransform.
     /// <include file='doc.xml' path='doc/members/member[@name="LightLDA"]/*' />
-    public sealed class LdaTransform : OneToOneTransformBase
+    public sealed class LdaTransform : OneToOneTransformerBase
```

Comment:
>LdaTransform [](start = 24, length = 12)

`LatendDirichletAllocationTransformer`

Reply:
For now I have renamed it as 'LdaTransformer' 
Using the suggested full name  fails the entry point unit test  . 

Cause : suggested full name has 36 characters.  However, `SetLoaderSig` has a check `Contracts.Check(sig.Length <= 24);`


---
In reply to: [232437634](https://github.com/dotnet/machinelearning/pull/1410#discussion_r232437634) [](ancestors = 232437634)

Reply:
thanks. makes sense now!

---
In reply to: [233581730](https://github.com/dotnet/machinelearning/pull/1410#discussion_r233581730) [](ancestors = 233581730,232469969,232437634)